tests/sentry/snuba/metrics/test_query_builder.py

```diff
@@ -174,7 +175,16 @@ def get_entity_of_metric_mocked(_, metric_name, use_case_id):
             'transaction:"/bar/:orgId/"',
             lambda: [
                 Condition(
-                    Column(name=resolve_tag_key(USE_CASE_ID, ORG_ID, "transaction")),
+                    Function(
+                        function="transform",
+                        parameters=[
+                            Column(name=resolve_tag_key(USE_CASE_ID, ORG_ID, "transaction")),
+                            [""]
+                            if options.get("sentry-metrics.performance.tags-values-are-strings")
```

Comment:
We don't need the option checks anymore

Reply:
Yep! I am fixing it rn because I developed it with still the option set (idk why but I could not default it to true on my machine, I might have missed something).